libraries/botbuilder-core/src/activityHandler.ts

```diff
@@ -397,6 +414,40 @@ export class ActivityHandler extends ActivityHandlerBase {
         await this.handle(context, 'Message', this.defaultNextEvent(context));
     }
 
+    /*
+     * Runs all registered _invoke_ handlers and then continues the event emission process.
+     * 
+     * @param context The context object for the current turn.
+     * 
+     * @remarks
+     * Overwrite this method to support channel-specific behavior across multiple channels.
+     * The default logic is to call any handlers registered via
+     * [onInvoke](xref:botbuilder-core.ActivityHandler.onInvoke),
+     * and then continue by calling [defaultNextEvent](xref:botbuilder-core.ActivityHandler.defaultNextEvent).
+    */
+    protected async onInvokeActivity(context: TurnContext): Promise<void|any> {
+        if(context.activity.name && context.activity.name === 'signin/verifyState' || context.activity.name === 'signin/tokenExchange') {
```

Comment:
define constants for `signin/verifyState` and `signin\tokenExchange` like we did in C#